src/app/shared/couchdb.service.ts

```diff
@@ -9,7 +9,11 @@ import { findDocuments, inSelector } from './mangoQueries';
 
 class DatePlaceholder {}
 
-@Injectable()
+@Injectable({
+  providedIn: 'root'
+})
+

```

Comment:
Please remove the extra added lines here

Reply:
@paulbert just finished removing extra line and the unused imports